probe/modules/database/nsrl/plugin.py

```diff
@@ -116,13 +116,15 @@ def __init__(self):
     # ==================
 
     def run(self, paths):
-        results = PluginResult(name=type(self).display_name,
-                               type=type(self).plugin_category,
+        results = PluginResult(name=type(self)._plugin_display_name_,
+                               type=type(self)._plugin_category_,
                                version=None)
         try:
             # lookup the specified sha1
             started = timestamp(datetime.utcnow())
-            response = self.module.lookup_by_sha1(sha1sum(paths).upper())
+            fileobj = open(paths,"r")
```

Comment:
Using fobj in the code i prefer to use context manager approach like

``` python
with open(paths, "r") as fileobj:
   response = self.module.lookup_by_sha1(sha1sum(fileobj))
```

Reply:
Thanks for your review.
I fixed to use context manager and squashed commits into one.